core/src/main/java/io/questdb/cutlass/text/CsvFileIndexer.java

```diff
@@ -120,9 +120,10 @@ public class CsvFileIndexer implements Closeable, Mutable {
     //if set to true then ignore first line of input file 
     private boolean header;
     private long lastQuotePos = -1;
-    private long errorCount = 0;
+    private int errorCount = 0;
```

Comment:
there can be an error on each line. so shouldn't errorCount have the same type as lineNumber?

Reply:
The thing is that error counters in other phases are `int`s, so that's why I decided to unify their types. An `int` is enough to hold errors for 2B rows. Do you think I should change all error counters to `long`?

Reply:
I think it would make sense. Either we can import >2B rows and then we should use long for error counters. 
or we cannot and then even the line counter should probably be int. if you mess-up a timestamp format (which is very easy) then it turns all rows into errors.

Reply:
Makes sense. Let me push a patch.

Reply:
Done in 3a5a33058286356eafd16f2ea321fb4b5c7f1067